app/presenters/finder_presenter.rb

```diff
@@ -23,6 +23,10 @@ def beta?
     content_item.details.beta
   end
 
+  def logo
+    content_item.details.logo
```

Comment:
Since you're just calling `content_item.details.logo` here, you can add it to the `delegate` section at the [top of the class](https://github.com/alphagov/finder-frontend/blob/add_eu_logo_to_esif_funds/app/presenters/finder_presenter.rb#L6).